package.json

```diff
@@ -3,14 +3,22 @@
   "version": "0.0.1",
   "private": false,
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.13.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.19.2",
     "eslint-config-airbnb-base": "^14.1.0",
+    "formik": "^2.1.4",
     "node-sass": "4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
```

Comment:
I'd love to know why you removed react-scripts

Reply:
I didn't realize react script was removed, I simply ran npm install